ConsoleApp1/ConsoleApp1.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <LangVersion>preview</LangVersion>
```

Comment:
what is this change? btw this commit message has the what but doesn't have the **why**

Reply:
I'd prefer to not use any preview features, I prefer that potential contributors can help using stable versions of Visual Studio